docs/guides/upgrading.rst

```diff
@@ -143,6 +147,9 @@ Schema changes
 The storage engine for the database tables has been changed from MyISAM to InnoDB. You maybe need to optimize your database settings for this change.
 The ``datalists`` table has been removed. All settings from datalists table have been merged into the ``config`` table.
 
+Metastrings in the database have been denormalized for performance purposes. We removed the metastrings table and put all the string values directly in the 
+metadata and annotation tables. You need to update your custom queries to reflect these changes. Also the ``msv`` and ``msn`` table aliases are no longer available.
```

Comment:
Could you add a general note that devs should not rely table aliases joined by ege* and do their own joins if they are implementing custom clauses.